app/views/keywords/_keywords_table.erb

```diff
@@ -12,17 +14,7 @@
 
     <tbody class="keywords-table__body">
       <% keyword_presenter.map do |presenter| %>
```

Comment:
Rails has built-in ways to render collections avoiding to use of `map`. See: `https://thoughtbot.com/blog/rendering-collections-in-rails`. In addition, to render collection, it is best to have a collection and item presenter. See https://github.com/nimblehq/mia-web/blob/development/engines/mia_campaign/app/presenters/mia_campaign/campaign_collection_presenter.r,  https://github.com/nimblehq/mia-web/blob/development/engines/mia_campaign/app/presenters/mia_campaign/campaign_presenter.rb and https://github.com/nimblehq/mia-web/blob/752f45a860b25bdc456c1bcc82b6b5a6ca18e334/engines/mia_admin/app/controllers/mia_admin/campaigns_controller.rb#L17.

So we could do the following:

```ruby
<%= keyword_list_presenter.keywords %>
```

Reply:
Fixed in 8cff1531c0df3753bdd71b255eaf1fbab8f09ee6